webdriver/tests/bidi/emulation/set_geolocation_override/invalid.py

```diff
@@ -214,7 +214,7 @@ async def test_params_coordinates_speed_invalid_type(bidi_session, top_context,
         )
 
 
-@pytest.mark.parametrize("value", [True, "foo", 42, {}])
+@pytest.mark.parametrize("value", [None, True, "foo", 42, {}])
```

Comment:
None of the equal tests for other commands actually use a `None` check here. If it's needed we should get all the tests updated (maybe even the `contexts` ones) in a separate PR.